From efc88b61330961965decc7a8d33310327c3306f1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Mon, 14 Oct 2019 20:30:35 +0200
Subject: [PATCH] Always add PROJECT_BINARY_DIR to include_directories

The directory is added by BoostTestTargets.cmake when Boost is found,
but due to a bug in the file it fails to detect boost even when it is
available.
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8a2f59f..3cb28ae 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -520,6 +520,7 @@ include(CheckCXXFeatures)
 # use BEFORE to ensure local directories are used first,
 # ahead of system-installed libs
 include_directories(BEFORE ${PROJECT_BINARY_DIR}/simgear)
+include_directories(BEFORE ${PROJECT_BINARY_DIR})
 
 add_definitions(-DHAVE_CONFIG_H)
 
-- 
2.23.0

